api/client/swarm/opts.go

```diff
@@ -86,40 +88,35 @@ func NewListenAddrOption() NodeAddrOption {
 
 // AutoAcceptOption is a value type for auto-accept policy
 type AutoAcceptOption struct {
-	values map[string]bool
+	values map[string]struct{}
 }
 
 // String prints a string representation of this option
 func (o *AutoAcceptOption) String() string {
 	keys := []string{}
-	for key, value := range o.values {
-		keys = append(keys, fmt.Sprintf("%s=%v", strings.ToLower(key), value))
+	for key := range o.values {
+		keys = append(keys, fmt.Sprintf("%s=%v", strings.ToLower(key), true))
```

Comment:
Okay, if you decide to keep the current format, can you change the Sprintf call to `fmt.Sprintf("%s=true", strings.ToLower(key))`?